examples/visualization/plot_evoked_arrowmap.py

```diff
@@ -0,0 +1,34 @@
+"""
+=============================================
+Plotting topographic arrowmaps of evoked data
+=============================================
+
+Load evoked data and plot arrowmaps along with
+the topomap for selected time points.
+Arrowmap is based upon Hosaka-Cohen transformation
```

Comment:
can you give a ref to this work? Also please use close to 80c lines.

Reply:
A new reference is coming in 3 weeks in Human Brain Mapping :)
I will put some old David reference